### ISSUE TITLE
Need a way to run example test locally

### ISSUE THREAD

**Issue by parameswarank** [robotframework/robotframework.github.com#269]
Hello,

I need to download the example code and run it locally. As of now i am unable to do it due to testobject issue.

**Link to Sample code:** [https://robotframework.org/code/?codeProject=N4IgdghgtgpiBcIDKBLKAHANjABAUQA9os4AaEAExgGcBjAJxXQBcUB7MBEEcgMxWzUEAbQC65KowBuMCgAV6bAFYxazBM3oBXGOUUAjNswBqMetXadEAVgB0ABlsBGEAF8gA](https://robotframework.org/code/?codeProject=N4IgdghgtgpiBcIDKBLKAHANjABAUQA9os4AaEAExgGcBjAJxXQBcUB7MBEEcgMxWzUEAbQC65KowBuMCgAV6bAFYxazBM3oBXGOUUAjNswBqMetXadEAVgB0ABlsBGEAF8gA)

Had discussed on Slack and i am asked to open up an issue on this.

Thanks
Parames

**Comment by emanlove** [comment]
I agree that all our examples and example code should be available so that someone can repeat these examples on their on systems.

**Comment by mkorpela** [comment]
A download button would be good for all the files.

**Comment by honglei** [comment]
https://robotframework.org/?tab=0#getting-started  is really confusing for I cannot find `from TestObject import TestObject` in CustomLibrary.py.
